packages/core/src/lib/lit-core.ts

```diff
@@ -469,6 +469,18 @@ export class LitCore {
   };
 
   /**
+   * returns the latest block hash.
+   * will call refresh if the block hash is expired
```

Comment:
JSDoc is misleading here -- this will _always_ return the latest blockhash from the nodes